docs/source/shared.rst

```diff
@@ -97,10 +98,11 @@ graph.
 Linear scaling with number of tasks
 ```````````````````````````````````
 
-As we increase the number of tasks in a graph we see that the scheduling
-overhead grows linearly.  The asymptotic cost per task depends on the
-scheduler.  The schedulers that depend on some sort of asynchronous pool have
-costs in the few milliseconds.  The schedulers that are single threaded are
+As we increase the number of tasks in a graph, we see that the scheduling
+overhead grows linearly.  The asymptotic (approaching a value or curve 
+arbitrarily closely) cost per task depends on the scheduler.  The schedulers 
```

Comment:
Yeah... "asymptotic" is obscure enough that it might be nice to add some sort of explanation, but "approaching a curve" doesn't really help get the reader to the idea that adding tasks is O(n). As you say, these are developers and probably have a solid grip on asymptotic cost, so I'll just remove the parenthetical comment.